src/types/types.ts

```diff
@@ -781,6 +781,22 @@ export interface NftAttributeRarity {
   prevalence: number;
 }
 
+/**
+ * Summary of the attribute prevalence for the specified NFT collection.
+ *
+ * @public
+ */
+export interface NftAttributesSummary {
+  /** The current NFT collection address. */
+  contractAddress: string;
+
+  /** The current NFT collection total supply. */
+  totalSupply: number;
+
+  /** The current NFT collection attribute prevalence summary grouped by trait type. */
+  summary: Record<string, Record<string, number>>;
```

Comment:
please modify comment to:
`The attribute prevalence of each trait grouped by the trait type for the provided NFT.`

Reply:
Considering that the address is for an NFT collection, should I add `collection` at the end of the prop description? The final text would be: `The attribute prevalence of each trait grouped by the trait type for the provided NFT collection`. I'll upload it as requested for the moment.

Reply:
Thanks! So to clarify, we want to standardize our wording to always to use NFT contract. Opensea has the concept of an "NFT collection" that can contain multiple different NFT contracts within it. So to avoid naming confusion, we want to clarify that this is for the individual contract, rather than the entire collection.

Reply:
Got it, thank you for the clarification! 🔥